docs/user/implement_algo.rst

```diff
@@ -348,9 +358,107 @@ For completeness, the full experiment file is repeated below:
 
     debug_my_algorithm()
 
+TensorFlow
+----------
+
+The TensorFlow version is similar to PyTorch except for building the
+computation graph and the :code:`train_once()` function.
+
+.. code-block:: python
+
+    import tensorflow as tf
+    import numpy as np
+
+    from garage import log_performance, TrajectoryBatch
+    from garage.sampler import RaySampler
+    from garage.misc import tensor_utils
+
```

Comment:
please make sure this is formatted in a way that would pass pre-commit and is actually tested.

an easy way to do this is to actually make this a file in `examples/` and then directly include it here. 

there's an example of how to do this in https://github.com/rlworkgroup/garage/pull/1765